### DISCUSSION TITLE
ok how do i execute this

### DISCUSSION THREAD

**Discussion by A5M1** [illsk1lls/ZipRipper#47] [Q&A]
i run as admin, i get to a launcher.zipripper file trying to open, i open and proceed to rereuun the script, and it opens gui, i try to select my 7z file and it says reg isnt a valid command when its part of windows

**Comment by A5M1** [comment]
![image](https://github.com/user-attachments/assets/ed2aca13-fa9e-462c-a4b7-be9a44e9861f)

**Comment by illsk1lls** [comment]
What version of windows are you running?

**Reply by A5M1** [reply]
> What version of windows are you running?

Windows 11 insider canary 27729.1000

**Comment by illsk1lls** [comment]
use windows explorer and double click the cmd file with the mouse

just as a heads up I am able to launch on my end with 23h2 in a powershell prompt using C:\Path\here> .\zipripper.cmd - but ZipRipper is a command script and it looks like you are trying to run it from inside WSL?

**Reply by A5M1** [reply]
> use windows explorer and double click the cmd file with the mouse
> 
> just as a heads up I am able to launch on my end with 23h2 in a powershell prompt using C:\Path\here> .\zipripper.cmd - but ZipRipper is a command script and it looks like you are trying to run it from inside WSL?

im inside regular pwsh7 i have msys in path so i can do ls, cat, etc and sudo is [native to w11](https://github.com/microsoft/sudo) now you just need to enable it
i figured it out however, closed with comment.

**Comment by A5M1** [comment]
Closing due to the fact it was an OS error, not a software bug.
